setup.py

```diff
@@ -474,11 +475,12 @@ def pxd(name):
                         'depends': (['pandas/_libs/src/klib/khash_python.h']
                                     + _pxi_dep['hashtable'])},
     '_libs.tslib': {'pyxfile': '_libs/tslib',
-                    'pxdfiles': ['_libs/src/util'],
+                    'pxdfiles': ['_libs/src/util', '_libs/lib'],
```

Comment:
did you add this back? ( I agree it *looks* like it should be here, but what was failing to indicate that)

Reply:
Looks like a screwup I made while rebasing.  Will remove momentarily.